app/src/main/java/com/mealsmadeeasy/ui/home/glance/WeekAtAGlanceMealViewHolder.kt

```diff
@@ -58,4 +64,40 @@ class WeekAtAGlanceMealViewHolder(
         })
         popup.show()
     }
+
+    private fun onClickEditServings() {
+        val root = View.inflate(itemView.context, R.layout.view_edit_servings, null)
+        val servingsText = root.findViewById<TextView>(R.id.add_to_plan_servings)
+        val plusButton = root.findViewById<ImageView>(R.id.add_to_plan_plus)
+        val minusButton = root.findViewById<ImageView>(R.id.add_to_plan_minus)
+
+        var numServings = mealPortion.servings
+        servingsText.text = numServings.toString()
+        minusButton.isEnabled = numServings != 1
+
+        minusButton.setOnClickListener {
+            if (numServings != 1) {
+                numServings--
+                servingsText.text = numServings.toString()
+                minusButton.isEnabled = (numServings > 1)
+            }
+        }
+
+        plusButton.setOnClickListener {
+            numServings++
+            minusButton.isEnabled = (numServings > 1)
+            servingsText.text = numServings.toString()
+        }
+
+
+        AlertDialog.Builder(itemView.context)
+                .setView(root)
```

Comment:
Let's give this a title that includes the name of the meal we're adjusting the servings for. If you want to use a string resource, For better translations, feel free to make the string resource have a value of `Edit servings for "%1$s"`. `getString()` also takes in vararg objects that fill in these placeholders, much like `sprintf` in C.